### PR TITLE
[indent] feat: indent multiple text blocks

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: packages/editor-common/web/src/Utils/draftUtils.js [hunk 1]
```diff
@@ -537,48 +538,40 @@ export function setSelection(editorState, selection) {
...
-  }
-
-  const content = editorState.getCurrentContent();
-  const block = content.getBlockForKey(key);
-  const depth = block.getDepth();
-
-  if (direction === 1 && depth === maxDepth) {
+  const contentState = editorState.getCurrentContent();
+  const startKey = selection.getStartKey();
+  const endKey = selection.getEndKey();
```
Comment (shaulgo) [inline packages/editor-common/web/src/Utils/draftUtils.js:557]: move into getBlocks...

File: packages/editor-common/web/src/Utils/draftUtils.js [hunk 2]
```diff
@@ -537,48 +538,40 @@ export function setSelection(editorState, selection) {
...
+  const blockMap = contentState.getBlockMap();
+  let blocks = getBlocksInSelection(blockMap, startKey, endKey);
```
Comment (shaulgo) [inline packages/editor-common/web/src/Utils/draftUtils.js:559]:
```suggestion
  let blocks = getBlocksInSelection(blockMap, startKey, endKey)
  .filter(block => isTypeText(block.getType());
```

File: packages/editor-common/web/src/Utils/draftUtils.js (hunk 1 above)
Comment (maimonav) [reply packages/editor-common/web/src/Utils/draftUtils.js:550]: changed.

File: packages/editor-common/web/src/Utils/draftUtils.js (hunk 2 above)
Comment (maimonav) [reply packages/editor-common/web/src/Utils/draftUtils.js]: changed.

**Pull request merged**